src/py/flwr/client/app.py

```diff
@@ -50,6 +51,19 @@ def run_client() -> None:
 
     args = _parse_args_client().parse_args()
 
+    # Obtain certificates
```

Comment:
@panh99 can we add a check here to fail if both `--insecure` and `--root-certificates` are set?

Reply:
Added.